### ISSUE TITLE
[SDK 2007] Pushing stuff with item_deliver + cooldown reset bug.

### ISSUE THREAD

**Issue by DoofusCockslap** [ZPSNMRiHIssues/zpsissues#214]
I will try to explain it step by step for better understanding since this issue is a very tricky stuff.
1. As the title says you can push the objects with item_deliver equipped i.e while it's active/selected in your inventory. You don't even have to press "H" to holster it.
    1) Select item_deliver
    2) Press mouse2
    3) There you go! You just pushed the prop without holstering a weapon.
    For some reason this applies only to item_deliver, other weapons are not affected by this.
2. Now this one is more interesting, you can reset a push cooldown using item_deliver.
    1) Hold "mouse2" (and keep it pressed)
    2) Select item_deliver
    3) Switch from item_deliver to any other weapon (using numbers of course).
    4) Switch back to item_deliver (Better to use "Q", which makes it easier).
    5) Keep repeatedly switching from/to item_deliver by pressing "Q" with specific timing.
3. There is more. Besides the item_deliver having this issue the phone also has it, but a little different (the phone does this slower).

Basically what the phone has, is the instant draw/holster animations which saves up your time and allows you to reset cooldown time, since it resets a cooldown any time you switch to "empty hands" with "H". Having this holster animation (or not having it at all) makes the phone holstering instant.

```
 1) Switch to the phone with "J"
 2) Then press "H" to holster phone. (by this, the phone won't switch back to other weapons with "J").
 3) Now when you will be pressing "J" again it will toggle weapon selection between "empty hands" and phone.
 4) Now you double tap "J" and mouse2. (Requires a good timing for this).
```

Here is the demonstration of how it works ingame:
https://youtu.be/OP-YvS7BwbU with

**Comment by jonnyboy0719** [comment]
This has now been fixed on the SDK 2013 build. _weapon_emptyhand_ never had any cooldown when you deployed it, which has been fixed now.
